change/@ni-nimble-components-568f1dc7-fe9b-4288-991a-c0d99cb4e256.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Is this PR associated with an issue that should be linked from the PR description?